src/stats.rs.in

```diff
@@ -0,0 +1,83 @@
+#[derive(Debug, PartialEq, Eq, Deserialize, Clone)]
+pub struct Period {
+    pub day: String,
+    pub views: u32,
+    pub visitors: u32,
+    pub reach: u32,
+    pub reach_subscribers: u32,
+    pub subscribed: u32,
+    pub unsubscribed: u32,
+    pub sex: Vec<Demography<DemoSex>>,
+    pub age: Vec<Demography<DemoAgeRange>>,
+    pub sex_age: Vec<Demography<DemoSexAgeRange>>,
+    pub cities: Vec<Demography<DemoCity>>,
+    pub countries: Vec<Country>,
+}
+
+enum_str! { DemoSex {
+    Male = "m",
+    Female = "f",
+}}
+
+enum_str! { DemoAgeRange {
+    _12_18 = "12-18",
+    _18_21 = "18-21",
+    _21_24 = "21-24",
+    _24_27 = "24-27",
+    _27_30 = "27-30",
+    _30_35 = "30-35",
+    _35_45 = "35-45",
+    _45_100 = "45-100",
+}}
+
+enum_str! { DemoSexAgeRange {
+    m_12_18 = "m;12-18",
+    m_18_21 = "m;18-21",
+    m_21_24 = "m;21-24",
+    m_24_27 = "m;24-27",
+    m_27_30 = "m;27-30",
+    m_30_35 = "m;30-35",
+    m_35_45 = "m;35-45",
+    m_45_100 = "m;45-100",
+    f_12_18 = "f;12-18",
+    f_18_21 = "f;18-21",
+    f_21_24 = "f;21-24",
+    f_24_27 = "f;24-27",
+    f_27_30 = "f;27-30",
+    f_30_35 = "f;30-35",
+    f_35_45 = "f;35-45",
+    f_45_100 = "f;45-100",
```

Comment:
No problem, I was thinking about  that. Just Implement from_str by hands. I will sync it tonight